Source/Locales/ORTS.Common/fr.po

```diff
@@ -194,846 +210,950 @@ msgctxt "Reverser"
 msgid "Control Electric Train Supply"
 msgstr "Contrôle Chauffage train"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:175
+#: ../../ORTS.Common/Input/UserCommand.cs:202
 msgid "Control Diesel Player"
 msgstr "Contrôle Locomotive diesel du joueur"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:176
+#: ../../ORTS.Common/Input/UserCommand.cs:203
 msgid "Control Diesel Helper"
 msgstr "Contrôle Locomotive diesel de pousse"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:177
+#: ../../ORTS.Common/Input/UserCommand.cs:204
 msgid "Control Headlight Increase"
 msgstr "Contrôle Augmentation de l'éclairage de tête"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:178
+#: ../../ORTS.Common/Input/UserCommand.cs:205
 msgid "Control Headlight Decrease"
 msgstr "Contrôle Diminution de l'éclairage de tête"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:179
+#: ../../ORTS.Common/Input/UserCommand.cs:206
 msgid "Control Injector 1 Increase"
 msgstr "Contrôle Augmentation de l'injecteur 1"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:180
+#: ../../ORTS.Common/Input/UserCommand.cs:207
 msgid "Control Injector 1 Decrease"
 msgstr "Contrôle Diminution de l'injecteur 1"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:181
+#: ../../ORTS.Common/Input/UserCommand.cs:208
 msgid "Control Injector 1"
 msgstr "Contrôle Activation/désactivation de l'injecteur 1"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:182
+#: ../../ORTS.Common/Input/UserCommand.cs:209
 msgid "Control Injector 2 Increase"
 msgstr "Contrôle Augmentation de l'injecteur 2"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:183
+#: ../../ORTS.Common/Input/UserCommand.cs:210
 msgid "Control Injector 2 Decrease"
 msgstr "Contrôle Diminution de l'injecteur 2"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:184
+#: ../../ORTS.Common/Input/UserCommand.cs:211
 msgid "Control Injector 2"
 msgstr "Contrôle Activation/désactivation de l'injecteur 2"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:185
+#: ../../ORTS.Common/Input/UserCommand.cs:212
 msgid "Control Blowdown Valve"
 msgstr "Contrôle Valve de drain"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:186
+#: ../../ORTS.Common/Input/UserCommand.cs:213
 msgid "Control Blower Increase"
 msgstr "Contrôle Augmentation de la soufflerie"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:187
+#: ../../ORTS.Common/Input/UserCommand.cs:214
 msgid "Control Blower Decrease"
 msgstr "Contrôle Diminution de la soufflerie"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:188
+#: ../../ORTS.Common/Input/UserCommand.cs:215
 msgid "Control Steam Heat Increase"
 msgstr "Contrôle Augmentation du chauffage du train"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:189
+#: ../../ORTS.Common/Input/UserCommand.cs:216
 msgid "Control Steam Heat Decrease"
 msgstr "Contrôle Réduction du chauffage du train"
 
-#: ../../ORTS.Common/Input/UserCommand.cs:190
+#: ../../ORTS.Common/Input/UserCommand.cs:217
+msgid "Control Steam Booster Air Valve"
+msgstr "Contrôle Soupape d'air de l'injecteur à vapeur"
```

Comment:
IIRC, the booster is some sort of "helper engine" that provides extra force. I don't know how that translates into French, but I don't think that "injector" is capturing the original meaning (but maybe I'm wrong).

Reply:
It's not really a compressor, just an auxiliary steam engine. I see that "booster" is accepted as a French word, so my suggestion is to translate it to something like "machine à vapeur booster" or even "moteur à vapeur renfort". I'm sure there must be a more accurate technical translation for it, but I couldn't find it.